sdk/log/logger_test.go

```diff
@@ -273,3 +273,28 @@ func TestLoggerEnabled(t *testing.T) {
 		})
 	}
 }
+
+func TestAllocationLimits(t *testing.T) {
+	const runs = 1
```

Comment:
```suggestion
	const runs = 10
```

Probably want more than 1 to ensure any initialization effect is not the only thing measured.

Reply:
Sorry, that is a typo 😂